controller/login_test.go

```diff
@@ -148,6 +162,68 @@ func (rest *TestLoginREST) TestLinkIdPWithTokenRedirects() {
 	test.LinkLoginTemporaryRedirect(t, svc.Context, svc, ctrl, nil, nil)
 }
 
+func (rest *TestLoginREST) TestResourceRequestPayload() {
+	t := rest.T()
+	resource.Require(t, resource.Database)
```

Comment:
This test is a remote test as well. We could specify both resource.Database and resource.Remote but it will mean that this test will be run for both make-remote and make-integration. I would rather use some mock Identity/User repo instead of real DB and move this test to a separate suite with resource.Require(t, resource.Remote)